task7/build.gradle

```diff
@@ -14,4 +14,9 @@ javadoc {
     classpath += sourceSets.test.compileClasspath
     source += sourceSets.test.allJava
     destinationDir = reporting.file("task7/build/docs/javadoc")
+}
+test {
```

Comment:
1. why do you exclude them?
2. better use `@Ignore` annotation on those classes?